wgsl/Makefile

```diff
@@ -1,24 +1,29 @@
 .PHONY: all clean
 
-all: index.html grammar/grammar.js test
+all: index.html grammar/grammar.js test wgsl.lalr.txt
```

Comment:
note: If you want to enable this on CI without enabling it for a default `make` invocation, you could add it to `compile.sh` instead

Reply:
Yeah, thanks.
I'll see if this bothers me or others.  Currently, the default target in wgsl also builds treesitter's WGSL grammar and checks examples. That already takes a few seconds, so I'm accustomed to manually running "make index.html" anyway.